### PR TITLE
Feature/bookmarkfunction

### PR DESCRIPTION
**Pull request by marcos-de-barros-vioque**
Hi!
In this PR I've included the following items: 
- Added a bookmark icon to the tips cards
- Created toggle bookmarking logic
- Created page for bookmarked tips
Thanks a lot in advance for checking this

### CODE REVIEW

File: pages/bookmarks/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        ) : (
+          ""
+        )}
+        {tips.map(
+          tip =>
+            tip.isBookmarked && (
+              <ListItem key={tip.id}>
+                <Tip
+                  tip={tip}
+                  toggleBookmarking={toggleBookmarking}
```
Comment (MerleMak) [inline pages/bookmarks/index.js:25]:
```suggestion
             onToggleBookmark={onToggleBookmark}
```

File: pages/bookmarks/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+export default function Bookmarks({tips, toggleBookmarking}) {
+  return (
+    <>
+      <StyledHeader>
+        <StyledTitle>Your Bookmarked Tips</StyledTitle>
+      </StyledHeader>
+      <StyledBody>
+        <h3>Here you can see the tips that you have bookmarked:</h3>
+        {!tips.find(tip => tip.isBookmarked) ? (
```
Comment (MerleMak) [inline pages/bookmarks/index.js:14]: are you sure you need the ! in fron of tips here?

**Review by MerleMak — COMMENTED** [review]
Hi Marcos, 
Nice work! I left some remarks concerning naming, but other than that looks great!
Keep it up 👍🏼☺️

File: pages/bookmarks/index.js (hunk 2 above)
Comment (marcos-de-barros-vioque) [reply pages/bookmarks/index.js:14]: Hi @MerleMak ! 

Thanks a bunch for checking this PR and sharing your feedback. 

I've just implemented changes for all pointed items, except for the last one (if I remove the "!" in front of the `tips.find` method, the text `You have not bookmarked any tip yet.` is not being displayed when no cards have been bookmarked).

Cheers!

**Pull request merged**